game-app/game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/AvailableSupports.java

```diff
@@ -149,8 +150,11 @@ int giveSupportToUnit(final Unit unit) {
     return amountOfSupportGiven;
   }
 
-  private int getSupportAvailable(final UnitSupportAttachment support) {
-    return Math.max(0, Math.min(support.getBonusType().getCount(), getSupportLeft(support)));
+  /** change to retrun the number of supporters (units) not supports */
```

Comment:
Comments should describe what the code is doing, not what it got changed to do from some other version of the code.

Reply:
Also, I don't know what the difference is between "supporters" and "supports" - both sound like they're talking about units. Is "supports" meant to mean something other than units? If so, we should explain.

Reply:
> > Also, I don't know what the difference is between "supporters" and "supports" - both sound like they're talking about units. Is "supports" meant to mean something other than units? If so, we should explain.
> 
> supporters are units, supports are the number of times this unit can give support.

Are they? I understood that "supporters" are actually supports (so not necessarily units unless in the game there are not support attachments sharing both the same type and the attached unit). That is, if I have two supports for the same unit (that is two support attachments attached to the same unit), they will be able to stack on the same target just as if they come from two different units, just, if they share the same type, they (or at least one of them if you are supporting different counts for the same type across different attachments) need a count greater than 1 for it to do so. I understood that only the case of a single support would never be applied more than once to the same target (regardless of its number, that defines the number of units which that support can support, and currently allows to apply it multiple times to a single unit if the count for the type is greater than 1). Am I not right?

Mind you that I'm not a developer and I've read none of the coding at this PR.

Reply:
> supports are the number of times this unit can give support

Ah! I think it's a very subtle definition, so I would actually suggest to spell it out (at least in comments).
i.e. when you want to talk about that concept, in comments, actually write "the number of times the unit can give support".